llap-tez/pom.xml

```diff
@@ -104,6 +104,11 @@
       <artifactId>hadoop-yarn-registry</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.xerial.snappy</groupId>
+      <artifactId>snappy-java</artifactId>
```

Comment:
Snappy and zstd are now optional on Avro, you probably have these already defined somewhere else but the tests in this module were complaining. I don't know if this could imply some runtime issue in other parts of the codebase that tests may not match, so worth to think about this if it is eventually the case.